tidb-cloud/recovery-group-get-started.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Get Started with Recovery Groups
+summary: Learn how to create a Recovery Group in TiDB Cloud and view its details
+---
+
+# Get Started with Recovery Groups
+
+This document describes how to create a Recovery Group to protect your databases running on TiDB Cloud Dedicated Clusters using the TiDB Cloud user interface. It also shows how to view details of the recovery group.
+
+## Prerequisites
```

Comment:
After Recovery Group is created, we will create account(in this pattern cloud-rg-*) in the secondary cluster, which will be used to replicated data into secondary cluster, we should remind user not to touch such account. If mistakenly delete such account, it will cause replication interrupted. 
I not should where to put this remind message, just comment here.